content/dstu2/openid-connect.md

```diff
@@ -0,0 +1,98 @@
+---
+title: OpenID Connect | DSTU 2 API
+---
+
+# Authorization Server OpenID Connect Support
+
+## Introduction
+
+[OpenID Connect][OPENID] describes itself as "a simple identity layer 
+on top of the OAuth 2.0 protocol".  The specification provides a set
+of message structures, a messaging protocol, and a security framework
+to allow a system that has authenticated a user to securely convey said
+identity to another service provider (relying party).  Cerner's 
+authorization server implements support for OpenID Connect in
+conjunction with the [SMART on FHIR<sup>®</sup>][SMART] profile 
+of OAuth 2.  Client applications may use this as a source of 
+"single sign-on" for users when protecting resources outside of the EMR 
+(example: user-generated data that is stored by the client application).
+
+In addition to SSO, SMART on FHIR<sup>®</sup> also leverages OpenID 
+Connect to convey the location of the authenticated user's 
+FHIR<sup>®</sup> endpoint, as described by the HL7<sup>®</sup>
+FHIR<sup>®</sup> standard.
+
+## Requesting OpenID Connect
+
+Per [OpenID Connect][OPENID] and [SMART on FHIR<sup>®</sup>][SMART], 
+a client application requests the scope of "openid" to receive an identity
+token within the access token response.
+
+## Identifying the User
+
+Upon retrieving an access token, an identity token will be presented in
```

Comment:
The previous documentation gave the sample response with both the access_token and id_token.  Can we provide an example like that?  Also, provide an example of a decrypted id_token with the iss and sub values?